src/lib/functions.ts

```diff
@@ -105,12 +107,13 @@ export async function trigger({
     }
 
     res?.send?.forEach((msg) => {
+      const isBroadcast = parsedAsyncAPI.server(msg.server).protocol() === 'ws' && !isRemoteServer(parsedAsyncAPI, msg.server)
```

Comment:
Only WebSocket and HTTP could be servers, and since currently, we are only supporting WebSocket, thus I am just checking if the message is being sent to a non-remote WebSocket server.

Reply:
What about this instead?

```suggestion
      const localServerProtocols = ['ws', 'wss', 'http', 'https']
      const serverProtocol = parsedAsyncAPI.server(msg.server).protocol().toLowerCase()
      const isBroadcast = localServerProtocols.includes(serverProtocol) && !isRemoteServer(parsedAsyncAPI, msg.server)
```